src/app/sitemap.ts

```diff
@@ -0,0 +1,17 @@
+import type { MetadataRoute } from "next";
+import { client } from "../../studio/lib/client";
+import { Slug } from "../../studio/lib/payloads/global";
+
+interface SitemapDocument {
+  slug: Slug;
+  _updatedAt: string;
+}
+
+export default async function sitemap(): Promise<MetadataRoute.Sitemap> {
+  const documents = await client.fetch<SitemapDocument[]>(`*[defined(slug)]`);
```

Comment:
this client fetches without a token — we're certain that's not needed?

Reply:
Good catch! This worked for me because my dataset was public. Fixed now to include the token.